frontend/packages/yki/src/components/registration/steps/SuomiFiRegistration.tsx

```diff
@@ -207,16 +198,20 @@ export const SuomiFiRegistration = ({
         label={t('termsAndConditions.label')}
         className="public-registration__grid__preview__privacy-statement-checkbox-label"
       />
-      <Text>
-        {t('privacyStatement.description')}:
-        <br />
-        <ExtLink
-          className="text-embed-link text-transform-none"
-          href={'endpoint'}
-          text={t('privacyStatement.linkLabel')}
-          endIcon={<OpenInNewIcon />}
-        />
-      </Text>
+      <div>
+        <Text>
+          {t('privacyStatement.description')}:
```

Comment:
EmailRegistration alla tää ainoana osana `Text` elementin alla. Jos seuraavassa pullarissa näitä yhdistelet niin ei pakko fiksaa jo täällä mutta yhtä hyvin vois